README.md

```diff
@@ -67,26 +67,28 @@ LavaMoat modifies the bundle's runtime to enforce the configured constraints.
 
 - [LavaMoat Browserify][LavamoatBrowserify]
 
-### Lavamoat Viz
+### lavamoat-viz
 
-[lavamoatViz]: https://lavamoat.github.io/LavaMoat/
+> [!WARNING] `lavamoat-viz` is currently unmaintained; we have tentative plans to resume development in the future.
 
-The [lavamoat viz][lavamoatViz] is a tool to visualize an application's dependency graph and assess package dangerousness.
+[lavamoatViz]: https://github.com/LavaMoat/LavaMoat/tree/lavamoat-viz
```

Comment:
(could) link to the branch where it's preserved

Reply:
that's what this is dogg

Reply:
brainfart